goty-server/src/main/kotlin/com/aleinin/goty/csv/CSVController.kt

```diff
@@ -1,14 +1,17 @@
 package com.aleinin.goty.csv
 
+import com.aleinin.goty.properties.PropertiesService
 import org.springframework.web.bind.annotation.CrossOrigin
 import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
 
 @CrossOrigin
 @RestController
 class CSVController(
-    private val csvService: CSVService
+    private val csvService: CSVService,
+    private val propertiesService: PropertiesService
 ) {
     @GetMapping(value = ["/csv"], produces = ["text/csv"])
-    fun getExport() = csvService.dumpToCSV()
+    fun getExport(@RequestParam(required = false) year: Int?) = csvService.dumpToCSV(year ?: propertiesService.getThisYear())
```

Comment:
While the server already had access to properties, I felt like defaults were more the job of the Controller than the Service, which is why i set the defaults at the Controller level (even though i needed to add PropertiesService to all the Controllers)